index.bs

```diff
@@ -440,6 +472,58 @@ To <dfn>maybe obtain an event attribution source</dfn> given an |anchor|, run th
 1. Round |source|'s [=attribution source/expiry=] away from zero to the nearest day.
 1. Return |source|.
 
+<h3 algorithm id="obtaining-attribution-source-params">Obtaining an attribution source from {{AttributionSourceParams}}</h3>
```

Comment:
Since having a shared algorithm seems desirable, we could probably just pass each data field as a parameter which seems fairly common.

The other option is to use AttributionSourceParams as the intermediate struct, which makes sense conceptually as the parameters should match across JS/HTML